classes/task/get_disk_usage.php

```diff
@@ -26,22 +26,35 @@ public function get_name() {
         return get_string('task_get_disk_usage', 'block_disk_quota');
     }
 
+
+//P.T. 29.09.2022 added conditions to have automatic site block optional ('block_site_enabled')
```

Comment:
this can be removed, people can search for the history with git blame if they need to see the date of the changes